### ISSUE TITLE
Usage mc_lib (v0.2) fails on windows

### ISSUE THREAD

**Issue by noDGodyaev** [ev-br/mc_lib#54]
I'm trying to build simple Cython app with mc_lib

tmp.pyx:
```
# language_level=3
from mc_lib.rndm cimport RndmWrapper
from mc_lib.observable cimport RealObservable
```

setup.py:
```
from distutils.core import setup
from distutils.extension import Extension
import numpy
from Cython.Build import cythonize
setup(
    ext_modules=cythonize("tmp.pyx"),
    include_dirs=[numpy.get_include()]
)
```
<details>
  <summary>Log</summary>
   
  ```running build_ext
building 'gen_samples_for_autocorr' extension
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -IC:\Users\dvgnn\anaconda3\envs\fast_p\lib\site-packages\numpy\core\include -IC:\Users\dvgnn\anaconda3\envs\fast_p\include -IC:\Users\dvgnn\anaconda3\envs\fast_p\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" /Tcgen_samples_for_autocorr.c /Fobuild\temp.win-amd64-3.7\Release\gen_samples_for_autocorr.obj
gen_samples_for_autocorr.c
C:\Users\dvgnn\anaconda3\envs\fast_p\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(23): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(23): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(23): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(25): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(31): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(31): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(31): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cstdlib(33): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\xtr1common(19): error C2061: синтаксическая ошибка: идентификатор "std"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\xtr1common(19): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\xtr1common(19): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\xtr1common(251): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(36): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(36): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(36): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(38): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(44): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(44): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(44): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(46): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(52): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(52): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(52): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(54): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(60): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(60): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(60): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(62): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(68): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(68): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(68): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(76): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(88): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(88): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(88): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(90): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(96): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(96): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(96): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(98): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(104): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(104): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(104): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(106): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(112): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(112): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(112): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(114): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(120): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(120): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(120): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(122): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(134): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(134): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(134): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(136): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(142): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(142): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(142): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(144): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(150): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(150): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(150): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(152): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(158): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(158): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(158): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(160): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(166): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(166): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(166): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(168): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(174): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(174): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(174): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(176): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(182): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(182): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(182): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(184): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(190): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(190): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(190): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(192): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(198): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(198): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(198): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(200): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(206): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(206): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(206): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(208): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(214): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(214): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(214): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(216): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(222): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(222): error C2059: синтаксическая ошибка: ;
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(222): error C2449: имеется "{" в пределах области видимости файла (возможно, отсутствует заголовок функции)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(224): error C2059: синтаксическая ошибка: }
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(230): error C2061: синтаксическая ошибка: идентификатор "noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include\cmath(230): fatal error C1003: число ошибок превышает 100; остановка компиляции

  ```
  
</details>

Win10
I'm using Anaconda env with
Python == 3.7
Cython == 0.29.24
numpy  == 1.21.2

Compiler: MCVC 14.29.30133

**Comment by ev-br** [comment]
Hm. 
Does a simple cython code compile? Cython + cimport numpy? Cython+observable? Cython+rndmWrapper?

**Comment by ev-br** [comment]
Also, is setup.py missing an include? https://github.com/ev-br/mc_lib/blob/master/examples/setup.py#L20

**Comment by noDGodyaev** [comment]
> Hm. Does a simple cython code compile? Cython + cimport numpy? Cython+observable? Cython+rndmWrapper?

It compiled without errors

> Also, is setup.py missing an include? https://github.com/ev-br/mc_lib/blob/master/examples/setup.py#L20

build with example `setup.py`:
```
import sys

from distutils.core import setup
from distutils.extension import Extension
from Cython.Distutils import build_ext

import numpy
import mc_lib



ext = Extension("tmp", ["tmp.pyx"],
                include_dirs = [numpy.get_include(),
                                mc_lib.get_include()],
                language='c++',
                )

setup(ext_modules=[ext],
      cmdclass = {'build_ext': build_ext})
```

get another problem
```
running build_ext
cythoning tmp.pyx to tmp.cpp
building 'tmp' extension
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -IC:\Users\dvgnn\anaconda3\envs\fast_p\lib\site-packages\numpy\core\include -IC:\Users\dvgnn\anaconda3\envs\fast_p\lib\site-packages\mc_lib -IC:\Users\dvgnn\anaconda3\envs\fast_p\include -IC:\Users\dvgnn\anaconda3\envs\fast_p\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" /EHsc /tmp.cpp /Fobuild\temp.win-amd64-3.7\Release\tmp.obj
tmp.cpp
C:\Users\dvgnn\anaconda3\envs\fast_p\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
tmp.cpp(649): fatal error C1083: Не удается открыть файл включение: _observable/observable.h: No such file or directory,
```

**Comment by ev-br** [comment]
Ok, that's saner. So something's up with the include paths. I'd try checking the path for a standalone c++ reproducer first.

**Comment by noDGodyaev** [comment]
There is no "_observable/observable.h" in "site-packages/mc_lib", could this result in an error?
```
 Содержимое папки C:\Users\dvgnn\anaconda3\envs\fast_p\Lib\site-packages\mc_lib

14.11.2021  20:32    <DIR>          .
14.11.2021  20:32    <DIR>          ..
14.11.2021  20:32    <DIR>          lattices
14.11.2021  20:32            70 144 observable.cp37-win_amd64.pyd
14.11.2021  20:32               725 observable.pxd
14.11.2021  20:32           153 088 rndm.cp37-win_amd64.pyd
14.11.2021  20:32               330 rndm.pxd
14.11.2021  20:32             1 663 setup.py
14.11.2021  20:32    <DIR>          tests
14.11.2021  20:32    <DIR>          __check_build
14.11.2021  20:32           156 160 __check_rndm.cp37-win_amd64.pyd
14.11.2021  20:32               473 __init__.py
14.11.2021  20:32    <DIR>          __pycache__
               7 файлов        382 583 байт
```

**Comment by ev-br** [comment]
Absolutely, yes! So, it just that stuff does not get installed. IIRC, I was only using in-place editable installs (pip install -e.). But if you figure out the install thing with the 0.2 setup, we'll happily tag it as 0.2.1

**Comment by noDGodyaev** [comment]
i found the solution.
In clean environment I've installed mc_lib via `pip install git+https://github.com/ev-br/mc_lib/releases/tag/v0.2" -- thats cause a problem.
In another env, I load 0.2v, then build it with `pip install -e .`, this method doesn't cause the error.

Checked both options on Ubuntu machine: both worked well.
In Google Colab  `pip install git+https://github.com/ev-br/mc_lib/releases/tag/v0.2` well.

Som this issue happend only on Windows, with one installation method.
